turbo.json

```diff
@@ -154,15 +154,15 @@
       "dependsOn": []
     },
     "lint": {
-      "cache": false,
+      "cache": true,
       "dependsOn": []
     },
     "lint:fix": {
       "cache": false,
```

Comment:
What's the reason for leaving this as false for my understanding?

Reply:
I just wanted to try and keep the changes minimal to task I know 100% benefit from caching. I'll be looking to add more in the future, but wanted to just enable caching gradually 👍🏼